src/pymedphys/dicom/_level2/header_tweaks.py

```diff
@@ -125,3 +126,40 @@ def adjust_rel_elec_density_cli(args):
         ds, adjustment_map, ignore_missing_structure=args.ignore_missing_structure)
 
     pydicom.write_file(args.output_file, new_ds)
+
+
+def RED_adjustment_map_from_structure_names(structure_names):
+
+    pattern = re.compile(
+        r'^.*RED\s*[=:]\s*(\d+\.?\d*)\s*$', flags=re.IGNORECASE)
```

Comment:
Could refactor out the regex and call it "structure_name_regex" for a little more explicitness?